packages/api-server/api_server/models/doors.py

```diff
@@ -1,12 +1,13 @@
 from pydantic import BaseModel, Field
+from tortoise.contrib.pydantic.creator import pydantic_model_creator
 
 from . import tortoise_models as ttm
 from .ros_pydantic import rmf_building_map_msgs, rmf_door_msgs
 
 Door = rmf_building_map_msgs.Door
 DoorMode = rmf_door_msgs.DoorMode
 DoorState = rmf_door_msgs.DoorState
-DoorHealth = ttm.DoorHealth
+DoorHealth = pydantic_model_creator(ttm.DoorHealth)
```

Comment:
Just to check, can this use `from .health import BasicHealth` like ingestors and fleet?

Reply:
this change is because the type checker/lint have problems with the old code, it doesn't infer the types correctly when using language features like class decorations.